app/views/components/_header-notice.html.erb

```diff
@@ -0,0 +1,39 @@
+<%
+  title ||= false
+  title_logo_url ||= false
+  heading ||= false
+  list ||= []
+  call_to_action ||= false
+  branding_classes ||= false
```

Comment:
We shouldn't be passing arbitrary classes to a component like this as it encourages restyling of restyling of components by applications, which breaks isolation. Instead there should be an option to enable a particular look for the component e.g. `nhs_style: true`.

Reply:
Ah yes - thanks for pointing this out. I’ll update this.